docs/Examples.rst

```diff
@@ -200,40 +242,151 @@ Again, we can easily write these equations for ``delicatessen``,
                 (data['Y'] - data['W']*theta[3])*(theta[0] - data['T'])
                 )
 
-    mestimator = MEstimator(psi, init=[0.1, 0.1, 0.1, 0.1])
-    mestimator.estimate()
+    estr = MEstimator(psi, init=[0.1, 0.1, 0.1, 0.1])
+    estr.estimate()
 
-This example further demonstrates the flexibility of M-estimation by stacking together estimating equations.
+    print(estr.theta)  # [-0.777, 0.005, 0.964, 1.769]
+
+This example demonstrates the flexbility of M-Estimation through the ability to stack estimating equations together.
 
 
 Robust Location (7.4.1)
 -------------------------------
 
-For robust location estimation, the estimating equation is
+To begin, we generate some generic data used for this example and several of the following
+
+.. code::
+
+    np.random.seed(7841)
+    y = np.random.normal(size=250)
+    n = y.shape[0]
+
+For the robust mean, the estimating equation proposed by Huber (1964) is
+
+.. math::
+
+    \psi_k(Y_i, \theta) = g_k(Y_i) - theta_1
+
+where :math:`k` indicates the bound, such that if :math:`Y_i>k` then :math:`k`, or :math:`Y_i<-k` then :math:`-k`,
+otherwise :math:`Y_i`. Below is the estimating equation translated into code
+
+.. code::
+
+    def psi_robust_mean(theta):
+        k = 3                          # Bound value
+        yr = np.where(y > k, k, y)     # Applying upper bound
+        yr = np.where(y < -k, -k, y)   # Applying lower bound
```

Comment:
yr is overwritten by itself, so example does not work as intended (error by ~0.02). 
Built-in estimating equation ([Line 144 of estimation_equations.py](https://github.com/pzivich/Delicatessen/blob/cb48d66b71fe606209683fb8f4d9d0875d53212a/delicatessen/estimating_equations.py#L144)) works as intended, so relative non-issue.